src/services/sync/strategies/consumerActions/ClassAction.js

```diff
@@ -90,14 +90,16 @@ class ClassAction extends BaseConsumerAction {
 		const teachers = [];
 		const ldapDns = !Array.isArray(uniqueMembers) ? [uniqueMembers] : uniqueMembers;
 
-		const users = await UserRepo.findByLdapDnsAndSchool(ldapDns, schoolId);
+		if (ldapDns[0]) {
```

Comment:
I think it should return here in case condition is not a valid DN to skip execute the extra update class with empty teacher and students

Reply:
Actually this is needed to fix current classes, which have been affected already